### DISCUSSION TITLE
Android Splash Screen

### DISCUSSION THREAD

**Discussion by salisuabubakar** [grahammendick/navigation#689] [Q&A] [answered]
Hi @grahammendick,
Is there any way i can do spalsh screen on android using The Navigation Router? If yes can you kindly give me some guidelines or documentation with example on how to do that.

 Thanks

**Comment by grahammendick** [comment, marked as answer]
No, this isn't part of the Navigation router. But you should be able to use any 3rd party splash screen package

**Reply by salisuabubakar** [reply]
ok Thanks.
